sdk/servicebus/service-bus/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@azure/service-bus",
   "sdk-type": "client",
   "author": "Microsoft Corporation",
-  "version": "7.0.0",
+  "version": "7.0.1",
```

Comment:
Interesting... why didnt the bot kick in after we released 7.0.0?
cc @HarshaNalluru, @richardpark-msft

Reply:
It looks like the pipeline failed during one of the steps to update the version number, however since the failure was in an unhandled promise exception, it didn't actually cause the job to fail...so we weren't notified about it :cry:

Reply:
Interesting.

@praveenkuttappan, do you know what's happening?